definitions/checks/check_hotfix_installed.rb

```diff
@@ -0,0 +1,87 @@
+class Checks::CheckHotfixInstalled < ForemanMaintain::Check
+  metadata do
+    label :check_hotfix_installed
+    description 'Check to verify if any hotfix installed on system'
+    preparation_steps do
```

Comment:
Shouldn't this be run during pre-upgrade checks? `tags :pre_upgrade`

Reply:
Yes, it is needed. updated!